__tests__/export.test.js

```diff
@@ -0,0 +1,64 @@
+import { getByTestId, getByText } from '@testing-library/testcafe';
+import { existsSync, readFileSync, unlinkSync } from 'fs';
+import { homedir } from 'os';
+import { join } from 'path';
+import omit from 'ramda/src/omit';
+import { ClientFunction } from 'testcafe';
+import timeLimit from 'time-limit-promise';
+
+import { injectLS } from './clientScripts';
+import { FIXTURE_LOCALSTORAGE_WITH_ONE_ACC, FIXTURES_CONST, PAGES } from './fixtures';
+import SettingsPage from './settings-page.po';
+import { getTransValueByKey } from './translation-utils';
+
+function getFilePath(fileName) {
+  return join(homedir(), 'Downloads', fileName);
+}
+
+async function waitForFile(path) {
+  while (!(await existsSync(path))) await setTimeout(() => {}, FIXTURES_CONST.TIMEOUT);
```

Comment:
Why awaiting on the `existsSync` synchronous call ?

Reply:
Indeed! Removed.